ui/karma.conf.js

```diff
@@ -3,16 +3,6 @@
 
 module.exports = function (config) {
   config.set({
-    customLaunchers: {
-      'webdriver': {
-        base: 'WebDriver',
-        browserName: 'chrome',
-        config: {
-          hostname: 'localhost',
-          port: 4444
-        },
-      }
-    },
```

Comment:
What does this chunk do? I seem to be able to run the test successfully without it.